### ISSUE TITLE
Failed to install truffle

### ISSUE THREAD

**Issue by alexbol99** [trufflesuite/truffle#4473]
I failed to install truffle on Windows 10
This is the first time I install truffle on this machine.
Please help

```
PS C:\Users\Alex Bol> npm install truffle -g
npm WARN deprecated testrpc@0.0.1: testrpc has been renamed to ganache-cli, please use this package from now on.
npm WARN deprecated mkdirp-promise@5.0.1: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated ipld-raw@6.0.0: This module has been superseded by the multiformats module
npm WARN deprecated apollo-tracing@0.15.0: The `apollo-tracing` package is no longer part of Apollo Server 3. See https://www.apollographql.com/docs/apollo-server/migration/#tracing for details
npm WARN deprecated circular-json@0.5.9: CircularJSON is in maintenance only, flatted is its successor.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated redux-devtools-instrument@1.10.0: Package moved to @redux-devtools/instrument.
npm WARN deprecated graphql-extensions@0.15.0: The `graphql-extensions` API has been removed from Apollo Server 3. Use the plugin API instead: https://www.apollographql.com/docs/apollo-server/integrations/plugins/
npm WARN deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated cids@1.1.9: This module has been superseded by the multiformats module
npm WARN deprecated ipld-dag-cbor@0.17.1: This module has been superseded by @ipld/dag-cbor and multiformats
npm WARN deprecated graphql-tools@6.2.6: This package has been deprecated and now it only exports makeExecutableSchema.\nAnd it will no longer receive updates.\nWe recommend you to migrate to scoped packages such as @graphql-tools/schema, @graphql-tools/utils and etc.\nCheck out https://www.graphql-tools.com to learn what package you should use instead
npm WARN deprecated remotedev-serialize@0.1.9: Package moved to @redux-devtools/serialize.
npm WARN deprecated redux-devtools-core@0.2.1: Package moved to @redux-devtools/app.
npm WARN deprecated apollo-cache-control@0.14.0: The functionality provided by the `apollo-cache-control` package is built in to `apollo-server-core` starting with Apollo Server 3. See https://www.apollographql.com/docs/apollo-server/migration/#cachecontrol for details.
npm WARN deprecated @nodefactory/filsnap-adapter@0.2.2: Package is deprecated in favour of @chainsafe/filsnap-adapter
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated multibase@3.1.2: This module has been superseded by the multiformats module
npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm WARN deprecated ipld-dag-pb@0.20.0: This module has been superseded by @ipld/dag-pb and multiformats
npm WARN deprecated multicodec@3.2.1: This module has been superseded by the multiformats module
npm WARN deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
npm WARN deprecated axios@0.20.0: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')
npm ERR! Cannot read properties of undefined (reading 'isServer')

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Alex Bol\AppData\Local\npm-cache\_logs\2021-11-29T19_09_22_117Z-debug.log
```
* Operating System: Windows 10
* Ethereum client: Geth 
* Truffle version (`truffle version`): none
* node version (`node --version`): 16.13.0
* npm version (`npm --version`): 8.1.4

**Comment by alexbol99** [comment]
Strange, but changing to this:
```
npm install -g truffle@5.4.22
```
actually helped to install
I closed this issue

**Comment by opleno** [comment]
Interesting... appending the version fixed the installation to me too, in Ubuntu 18.04.
